src/main/java/com/emc/ecs/nfsclient/network/ClientIOHandler.java

```diff
@@ -81,7 +81,9 @@ public InetSocketAddress getRemoteAddress() {
      * org.jboss.netty.channel.ChannelStateEvent)
      */
     public void channelConnected(ChannelHandlerContext ctx, ChannelStateEvent e) throws Exception {
-        LOG.info("Connected to: {}", getRemoteAddress());
+        if (LOG.isDebugEnabled()) {
```

Comment:
There's no need to check for debug... the overhead is minimal in slf4j due to template strings